internal/auth/auth.go

```diff
@@ -24,6 +24,24 @@ import (
 	storagev1 "google.golang.org/api/storage/v1"
 )
 
+const universeDomainDefault = "googleapis.com"
+
+func getUniverseDomain(ctx context.Context, contents []byte, scope string) (string, error) {
+	creds, err := google.CredentialsFromJSON(ctx, contents, scope)
+	if err != nil {
+		err = fmt.Errorf("Error in getting credentials: %v", err)
```

Comment:
Let's return the error by adding prefix of method name. Also use %w in most of the cases.

err = fmt.Errorf("CredentialsFromJSON(): %w", err)
Similarly for other places.